addons/account/wizard/accrued_orders.py

```diff
@@ -155,9 +155,10 @@ def _ellipsis(string, size):
         fnames = []
         total_balance = 0.0
         for order in orders:
-            if len(orders) == 1 and self.amount and order.order_line:
+            product_lines = order.order_line.filtered(lambda x: x.product_id)
+            if len(orders) == 1 and len(product_lines) > 0 and self.amount and order.order_line:
```

Comment:
```suggestion
            if len(orders) == 1 and product_lines and self.amount and order.order_line:
```
Cleaner

Reply:
Thanks for the suggestion! I've made the change :)